src/ResourceManager/Sql/Commands.Sql/Commands.Sql.csproj

```diff
@@ -446,8 +454,8 @@
       <HintPath>..\..\..\packages\Microsoft.Azure.Management.Sql.1.3.0-preview\lib\net452\Microsoft.Azure.Management.Sql.dll</HintPath>-->
     <Reference Include="Microsoft.Azure.Management.Sql, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Sql.1.7.0-preview\lib\net452\Microsoft.Azure.Management.Sql.dll</HintPath>
-      <Private>True</Private>
+      <HintPath>..\..\..\..\tools\LocalFeed\Microsoft.Azure.Management.Sql.1.7.0-preview\lib\net452\Microsoft.Azure.Management.Sql.dll</HintPath>
```

Comment:
@maboja-msft just a reminder to point at version 1.8.0-preview when it is published to NuGet